Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:2-slim
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user && mkdir -p /code && chown user:user /code
+
+RUN apt-get update && apt-get install -y --no-install-recommends \

```

Comment:
Right, this is just because I started the dockerization before thinking about forking the repo. If the Dockerfile is integrated in the repo (and published automatically to Docker Hub ?) the COPY is far more better and lighter.